src/constants/Routes.ts

```diff
@@ -0,0 +1,25 @@
+import { RouteObject } from '../types/index';
+
+import Home from 'pages/Home';
+import Styleguide from 'pages/Styleguide';
+
+const RouteMap: { [id: string]: RouteObject } = {
+  HOME: {
+    path: '/',
+    component: Home,
+    exact: true,
+  },
+  STYLEGUIDE: {
+    path: '/styleguide',
+    component: Styleguide,
+    exact: true,
+    dev: true,
+  },
+};
+
+export const ROUTES =
+  process.env.NODE_ENV === 'development'
```

Comment:
It looks like you are already doing this in `lib/config`, I would suggest import `Config` and just doing `Config.DEVELOPMENT ? etc..`